### PR TITLE
[13.0][MIG] account_move_line_manufacture_info

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module account_move_line_manufacture_info to v13.

cc ~ @ForgeFlow

### CODE REVIEW

File: account_move_line_manufacture_info/__manifest__.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# © 2019 Eficent Business and IT Consulting Services S.L.
+# © 2019 ForgeFlow S.L.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Account Move Line Manufacture Information",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "depends": ["stock_account", "mrp"],
-    "author": "Eficent," "Odoo Community Association (OCA)",
+    "author": "ForgeFlow," "Odoo Community Association (OCA)",
```
Comment (LoisRForgeFlow) [inline account_move_line_manufacture_info/__manifest__.py]: remove intermediate quotes

**Review by LoisRForgeFlow — APPROVED** [review]
Code review looks good :+1:, a minor comment:

File: account_move_line_manufacture_info/__manifest__.py (hunk 1 above)
Comment (AdriaGForgeFlow) [reply account_move_line_manufacture_info/__manifest__.py]: :+1:

**Review by HviorForgeFlow — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**